server.go

```diff
@@ -128,8 +126,12 @@ type Server struct {
 	GRPCListenAddrs   []net.Addr
 	RcvbufBytes       int
 
-	interval            time.Duration
-	synchronizeInterval bool
+	enableMetricRouting      bool
+	computationRoutingConfig []ComputationRoutingConfig
+	interval                 time.Duration
+	synchronizeInterval      bool
+	lastFlushes              map[string]*int64
```

Comment:
Again, we key by Metrics Computation Routing name - I'm not sure if this is the cleanest way to do it and am open to ideas.